src/app/components/Curation/getComponentName/index.test.ts

```diff
@@ -28,9 +30,24 @@ describe('getComponentName', () => {
     ${RANKED}       | ${NORMAL}            | ${MOST_READ}
     ${'fake-style'} | ${'fake-prominence'} | ${null}
   `(
-    'should return $expected when visual style is $visualStyle and visual prominence is $visualProminence',
+    'should return $expected when visual style is $visualStyle and visual prominence is $visualProminence and curation type is $curationType',
     ({ visualStyle, visualProminence, expected }) => {
-      expect(getComponentName(visualStyle, visualProminence)).toBe(expected);
+      expect(
+        getComponentName({
+          visualStyle,
+          visualProminence,
+          radioSchedule: undefined,
+        }),
+      ).toBe(expected);
     },
   );
+  it('should return radio schedule when curationType is radio-schedule and visual style is NONE and visualProminence is NORMAL', () => {
```

Comment:
Let's update the name of this test 
```suggestion
  it('should return radio schedule when a radio schedule is present', () => {
```

Reply:
done